dnssec_keyscan.go

```diff
@@ -166,6 +171,39 @@ func readPrivateKeyECDSA(m map[string]string) (*ecdsa.PrivateKey, error) {
 	return p, nil
 }
 
+func readPrivateKeyED25519(m map[string]string) (ed25519.PrivateKey, error) {
+	var p ed25519.PrivateKey
+	// TODO: validate that the required flags are present
```

Comment:
Should we do this TODO here now as well? Or just garbage in garbage out?

Reply:
@miekg I think there is a compelling argument to be strict from the get go, but at the same time this is consistent with all the other key types so I'd be inclined to leave it. It could be worth making them all stricter in a separate pull request though.